src/components/Home/Shop.js

```diff
@@ -1,17 +1,25 @@
 import { StarIcon } from '@heroicons/react/solid'
-import React from 'react'
+import React, { useCallback } from 'react'
 import { Link } from 'react-router-dom'
 
-function Shop ({ name, address }) {
+function Shop ({ address, loading, name }) {
```

Comment:
Please add prop type validation for this component. Like in Auxilliary component.